assets/dev/scss/frontend/widgets/link-in-bio.scss

```
@@ -100,34 +176,52 @@
 			border-color: var(--e-link-in-bio-identity-image-cover-border-color);
 			border-style: var(--e-link-in-bio-identity-image-cover-border-style);
 			border-width: 0 0 var(--e-link-in-bio-identity-image-cover-border-bottom-width) 0;
-			margin-block-start: calc(var(--e-link-in-bio-gutter) * -1);
-			margin-inline: calc(var(--e-link-in-bio-gutter) * -1);
+			margin-block-start: calc(var(--e-link-in-bio-gutter-block-start) * -1);
+			margin-inline: calc(var(--e-link-in-bio-gutter-inline) * -1);
 			max-height: var(--e-link-in-bio-identity-image-cover-height);;
-			width: calc(100% + calc(var(--e-link-in-bio-gutter) * 2));
+			width: calc(100% + calc(var(--e-link-in-bio-gutter-inline) * 2));
 
 			&.has-border {
 				--e-link-in-bio-identity-image-cover-border-style: solid;
 			}
 		}
 	}
 
+	&__bio {
+
+		> * {
+			margin-block: 0;
+		}
+	}
+
 	&__heading {
 		color: var(--e-link-in-bio-heading-color);
+		font-size: 36px;
+		font-weight: 600;
+		line-height: 42px;
 	}
 
 	&__title {
 		color: var(--e-link-in-bio-title-color);
+		font-size: 20px;
+		font-weight: 500;
+		line-height: 35px; // 20px in Figma is not accurate
```

```suggestion
		line-height: 35px;
```
You can remove this comment

---

done